GTG/gtk/data/main_window.ui

```diff
@@ -1,59 +1,61 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Generated with glade 3.36.0 -->
+<!-- Generated with glade 3.38.2 -->
```

Comment:
Possibly "remove" glade introduced automatic changes so reviewing is a bit easier (to see what has been changed).

Reply:
Removed a bit, but it's hard to tell what can be removed from the auto changes without breaking things